### ISSUE TITLE
[Topic] Keyboard navigation testing

### ISSUE THREAD

**Issue by rianrietveld** [wpaccessibility/wp-a11y-docs#175]
URL: https://wpaccessibility.org/docs/testing/keyboard/
Code: https://github.com/wpaccessibility/wp-a11y-docs/blob/main/docs/testing/keyboard/index.md

Please read [How to contribute](https://wpaccessibility.org/docs/contribute/) before you start.

## Tasks

This content should be reviewed expanded.
The links need to be checked and updated.
